src/hooks/transactions/helpers/isOlderThan.ts

```diff
@@ -0,0 +1,7 @@
+export const isOlderThan = (batchId: string, olderThanMs: number) => {
```

Comment:
add a comment saying what does this do
I would not send a batchId here. I would send 2 numbers, and call it `timestampIsOlderThan`